src/framework/theme/components/accordion/accordion-item-body.component.ts

```diff
@@ -4,15 +4,7 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import {
-  Component,
-  ChangeDetectionStrategy,
-  Host,
-  ElementRef,
-  ChangeDetectorRef,
-  OnInit,
-  OnDestroy,
-} from '@angular/core';
+import { Component, ChangeDetectionStrategy, Host, ChangeDetectorRef, OnInit, OnDestroy } from '@angular/core';
```

Comment:
is this necessary?

Reply:
it's not important, but more readable now.

Reply:
:) it's debatable, I suggest we not mix code style changes into the particular issue fixes

Reply:
Fixed.